server/src/controllers/job_posts/getAllJobPosts.js

```diff
@@ -0,0 +1,85 @@
+// Get All Job Posts
+// a) GET request to get all job posts [id, title, employer, application_close_date, state]
+// Return the id, title, employer, application_close_date, state values of ALL job posts
+
+// b) GET request to get all public job posts 
+// Return the id, title, employer, location, application_close_date of job posts where state is "Active"
+
+// c) GET request by job type using job type parameter
+// Return all job posts with that job type and state = "Active"
+
+const { query } = require("express");
```

Comment:
I think query is a part of the 'req' object and doesn't need to be explicitly imported. However, it's fine to leave it or remove it!